### ISSUE TITLE
TigerVNC Viewer does not resize window in Windows 10

### ISSUE THREAD

**Issue by fcying** [TigerVNC/tigervnc#425]
OS: Windows 10 Pro
TigerVNC Viewer  1.7.1
TigerVNC Server  1.7.1

**Comment by CendioOssman** [comment]
I'm not sure what you mean when you say it doesn't resize the window? It doesn't support changing resolution on the Windows server, if that's what you mean?

**Comment by fcying** [comment]
yes, It doesn't support changing resolution on the Windows server. 

I can still see the scroll bars.

**Comment by skyshore2001** [comment]
it's quite inconvenient when I connect to my desktop via my laptop, that the remote resolution is bigger than my laptop, and I have to scroll the window always. I see it's hard to ask the server-side to change resolution, however, I will appreciate if there's a feature option shrink the remote desktop to proper size. 

I search solution in the internet and find tightvnc has a "Scale by" option with value "Auto", that is exactly meet my requirement! Hope tigervnc can adopt it.

**Comment by CendioOssman** [comment]
Please see #274 for client-side scaling. It also has suggestions for alternatives.
